vignettes/quitte-data-analysis.Rmd

```diff
@@ -234,23 +239,23 @@ or across other dimensions
 
 ```{r sum variables}
 data %>% 
-    filter('2°C' == scenario, 
-           'RUS' == region,
-           2035 == period,
+    filter(scenario == '2°C', 
```

Comment:
non-ASCII symbols are not allowed in our packages. So this line would not be accepted there, On the other hand this is also true for the NSE you are using in all examples.

Reply:
> non-ASCII symbols are not allowed in our packages

That is true for R code, but the vignette is technically documentation, so different rules apply.  Which is the reason `R CMD build` does not complain about it.